modules/80_optimization/nash/postsolve.gms

```diff
@@ -304,16 +304,17 @@ loop(regi,
      );
 ); !!regi
 
-***additional criterion: are the anticipation terms sufficienctly small?
+*** criterion only for checking, not applied anymore: are the anticipation terms sufficienctly small?
 p80_fadeoutPriceAnticip_iter(iteration) = sm_fadeoutPriceAnticip;
 if(sm_fadeoutPriceAnticip gt cm_maxFadeOutPriceAnticip, 
-  s80_bool = 0;
+***  s80_bool = 0; !! not an active convergence criterion anymore 
   p80_messageShow("anticip") = YES;
 );
+
 *' criterion "Deviation due to price anticipation": are the resulting deviations sufficiently small?
 *' compare to 1/10th of the cutoff for goods imbalance 
 if(p80_DevPriceAnticipGlobAllMax("2100") gt 0.1 * p80_surplusMaxTolerance("good"),
```

Comment:
for the plot we need a parameter which depends on iterations adn we should use the same parameter for teh plot which is used in the criterion

Reply:
Lavinia, there is a corresponding parameter with an iteration index: p80_DevPriceAnticipGlobAllMax2100Iter
right?
Looks fine for me.

Reply:
ifso, we should use `p80_DevPriceAnticipGlobAllMax2100Iter` here for the criterion

Reply:
done